packages/components/select/src/select-option-group.ts

```diff
@@ -7,11 +8,14 @@ export class SelectOptionGroup extends LitElement {
   /** @private */
   static override styles: CSSResultGroup = styles;
 
+  /** Select size. */
+  @property({ reflect: true }) size: SelectSize = 'md';
+
   @property({ attribute: 'group-title' }) groupTitle?: string;
```

Comment:
Wouldn't it be more consistent if this were called `group-heading`/`groupHeading`? In other places I try to avoid `title` since that triggers a native tooltip.